<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>******菜单******</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_WK">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密钥注充</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM2E">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM2加密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM4D">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM4解密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM4E">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM4加密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM2D">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM2解密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM2S">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM2签名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SM2V">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SM2验签</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
